src/rqa/histograms.jl

```diff
@@ -100,9 +100,9 @@ function _linehistograms(rows::T, cols::T, lmin::Integer, theiler::Integer,
 end
 
 deftheiler(x::Union{RecurrenceMatrix,JointRecurrenceMatrix}) = 1
-deftheiler(x::CrossRecurrenceMatrix) = 0
+deftheiler(x::Union{CrossRecurrenceMatrix,SparseMatrixCSC}) = 0
```

Comment:
Not a good idea to group the default of a `CrossRecurrenceMatrix` with that of a `SparseMatrix`. Please add a third definition. Also, givcen that we do not know the background of the sparse matrix we should use the safer option and give it a default theiler of 1.